website/src/components/SearchPage/fields/DateRangeField.tsx

```diff
@@ -0,0 +1,131 @@
+    }
+}
+
+export const DateRangeField = ({ field, fieldValues, setSomeFieldValues }: DateRangeFieldProps) => {
+    // the DateRangeField expects 4 fields in groupedFields and they should all have rangeOverlapSearch defined
+    const getField = (bound: string, fromTo: string) =>
+        field.groupedFields.filter((f) => f.name.endsWith(fromTo) && f.rangeOverlapSearch!.bound === bound)[0];
+    const lowerFromField = getField('lower', 'From');
+    const lowerToField = getField('lower', 'To');
+    const upperFromField = getField('upper', 'From');
+    const upperToField = getField('upper', 'To');
+
+    const [strictMode, setStrictMode] = useState(
+        isStrictMode(
+            lowerFromField.name in fieldValues,
+            lowerToField.name in fieldValues,
+            upperFromField.name in fieldValues,
+            upperToField.name in fieldValues,
+        ),
+    );
+
+    const lowerField = strictMode ? lowerFromField : upperFromField;
+    const upperField = strictMode ? upperToField : lowerToField;
+    const [lowerValue, setLowerValue] = useState(fieldValues[lowerField.name] ?? '');
+    const [upperValue, setUpperValue] = useState(fieldValues[upperField.name] ?? '');
+
+    useEffect(() => {
+        if (strictMode) {
+            setSomeFieldValues(
+                [lowerFromField.name, lowerValue],
+                [upperToField.name, upperValue],
+                [upperFromField.name, null],
+                [lowerToField.name, null],
+            );
+        } else {
+            setSomeFieldValues(
+                [upperFromField.name, lowerValue],
+                [lowerToField.name, upperValue],
+                [lowerFromField.name, null],
+                [upperToField.name, null],
+            );
+        }
+    }, [
+        strictMode,
+        lowerValue,
+        upperValue,
+        lowerFromField,
+        lowerToField,
+        upperFromField,
+        upperToField,
+        setSomeFieldValues,
+    ]);
+
+    return (
+        <div key={field.name} className='flex flex-col border p-3 mb-3 rounded-md border-gray-300'>
+            <div className='flex flex-row justify-between items-baseline mb-2'>
+                <h3 className='text-gray-500 text-sm'>{field.displayName}</h3>
+                <CustomTooltip id={'strict-tooltip' + field.name}>
+                    <div className='w-52'>
+                        <p>
+                            <span className='font-bold'>strict: </span>
+                            {field.displayName} range must be <span className='italic'>entirely </span>
+                            inside of the search range.
+                        </p>
+                        <p>
+                            <span className='font-bold'>not strict: </span>
+                            {field.displayName} range must have <span className='italic'>some overlap </span>
+                            with the search range.
```

Comment:
I think that it would be great to have some description about this in the [search sequences documentation](https://loculus.org/for-users/search-sequences/). The tooltips are already really nice but on the documentation page, we can a bit more precise and maybe have an example?

Reply:
https://docs-date-ranges-ui.loculus.org/for-users/search-sequences/

(Image not loading yet, maybe just not deployed yet?)

Reply:
Nice! Looks amazing!